5.5/Dockerfile.rhel7

```diff
@@ -0,0 +1,20 @@
+FROM rhel7
+
+ADD mysql /opt/openshift/
+ENV PATH /opt/openshift/bin:$PATH
+
+WORKDIR /opt/openshift
+RUN create_user.sh
+
+RUN yum install -y yum-utils hostname && \
+    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
+    yum-config-manager --enable rhel-7-server-optional-rpms && \
+    yum install -y --setopt=tsflags=nodocs mysql55 && \
+    yum clean all
+
+VOLUME ["/var/lib/mysql"]
+
+ENTRYPOINT ["entrypoint.sh"]
```

Comment:
entrypoint.sh is from upstream mysql image. While `run-scl-mysql.sh` is more descriptive, I dislike dropping "entrypoint" from the name, because that makes it clear that the script serves as an entrypoint. Maybe `entrypoint-scl-mysql.sh`?